<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EnregistrementMessageDialog</class>
 <widget class="QDialog" name="EnregistrementMessageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Enregistrement de message</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QGroupBox" name="typeMessageGroupBox">
       <property name="title">
        <string>Type de message</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QRadioButton" name="promotionRadio">
            <property name="text">
             <string>Promotion</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="pointAppelRadio">
            <property name="text">
             <string>Point appel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="ouvertureRadio">
            <property name="text">
             <string>(Pré) Ouverture</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QRadioButton" name="programmesRadio">
            <property name="text">
             <string>Programmés</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="jingleRadio">
            <property name="text">
             <string>Jingle</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="fermetureRadio">
            <property name="text">
             <string>(Pré) Fermeture</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="mainLayout">
       <item>
        <layout class="QHBoxLayout" name="titreLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Titre du message :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="titreLabel">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="changeNomButton">
           <property name="text">
            <string>Nouveau titre</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="recorderLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>TODO : zone enregistrement</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="buttonLayout">
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="playButton">
           <property name="text">
            <string>Jouer</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="saveButton">
           <property name="text">
            <string>Sauvegarder</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="supprimerButton">
           <property name="text">
            <string>Supprimer</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="closeLayout">
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="closeButton">
         <property name="text">
          <string>Fermer</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>29</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>promotionRadio</tabstop>
  <tabstop>pointAppelRadio</tabstop>
  <tabstop>ouvertureRadio</tabstop>
  <tabstop>programmesRadio</tabstop>
  <tabstop>jingleRadio</tabstop>
  <tabstop>fermetureRadio</tabstop>
  <tabstop>changeNomButton</tabstop>
  <tabstop>playButton</tabstop>
  <tabstop>saveButton</tabstop>
  <tabstop>supprimerButton</tabstop>
  <tabstop>closeButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>EnregistrementMessageDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>247</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
